.github/workflows/self-hosted.yml

```diff
@@ -74,32 +74,13 @@ jobs:
           python3 -m tools.fast_editable --path .
           python3 -m sentry.build.main
 
-      - run: docker login --username '${{ github.actor }}' --password '${{ github.token }}' ghcr.io
-        if: github.ref_name == 'master' && github.event_name != 'pull_request'
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # v3.10.0
-
-      - name: Build
-        uses: docker/build-push-action@32945a339266b759abcbdc89316275140b0fc960 # v6.8.10
+      - uses: getsentry/action-build-and-push-images@d129d6a730b62e258c730618ba072d605dadc798
```

Comment:
Picks up https://github.com/getsentry/action-build-and-push-images/pull/8 that should resolve the failure here:

https://github.com/getsentry/sentry/actions/runs/16729537407/job/47353977258